tests/unit_tests/commandline/test_log_setup.py

```diff
@@ -151,6 +151,7 @@ def test_log_with_emphasis_long_word(message, caplog):
     log_with_emphasis(LOG.info, message)
     log_lines = caplog.messages[1 : len(caplog.text) - 1]
 
+    print(f"LOG LINES = {log_lines}")
```

Comment:
Are we wanting to leave this in?

Reply:
I removed that. Caught that right after I pushed